app/src/main/res/layout/activity_migration.xml

```diff
@@ -33,7 +34,7 @@
                 android:layout_marginStart="24dp"
                 android:maxLines="2"
                 android:text="@string/migration_title"
-                android:fontFamily="sans-serif-medium"
+                android:fontFamily="Sharp Sans"
```

Comment:
I have never seen this font name. Is that a font we ship with Fenix? @ekager @sblatz maybe you can review this change?

Reply:
I've found this font in use in `fragment_crash_reporter.xml`. That's why I took the liberty to use it again.

Reply:
@AmyYLee what font should we be using here?